src/components/Profile.vue

```diff
@@ -28,7 +28,8 @@ export default {
     methods: {
         onLogout: function () {
             Store.unAuthenticate();
-            this.$router.push("/login");
+            const routeObj = { name: "login", params: { logoutReason: 200 } };
```

Comment:
what does logoutReason 200 mean? Could be made more clear here that it means "normal logout"

Reply:
I'll add a comment